PollinatorPathway/Controllers/ImageAPI.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using PollinatorPathway.Data;
+using PollinatorPathway.Model;
+
+namespace PollinatorPathway.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ImageAPI : ControllerBase
```

Comment:
Please rename this class to "Images" so that the  access URL will better match the convention
`https://localhost:7221/api/images`